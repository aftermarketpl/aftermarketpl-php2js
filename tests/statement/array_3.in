$a = [2, 3, 4];
return $a;
