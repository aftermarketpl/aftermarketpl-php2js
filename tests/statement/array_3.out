var a;
a = [
    2,
    3,
    4
];
return a;
